packages/core-data/src/selectors.js

```diff
@@ -104,7 +104,20 @@ export function getEntity( state, kind, name ) {
  * @return {Object?} Record.
  */
 export function getEntityRecord( state, kind, name, key ) {
```

Comment:
This selector is now returning a new object on each call, it should be memoized.

Reply:
Good catch, here:

https://github.com/WordPress/gutenberg/pull/16932/commits/f2c2c990f76efd8bae039f31be17fd29961d338c